config/migrations/20230717221117-add-secretaris-role.ttl

```diff
@@ -0,0 +1,22 @@
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix thConceptScheme: <http://themis.vlaanderen.be/id/concept-scheme/> .
+
+thConceptScheme:b18acf1a-2a37-4b42-a549-b158d0065092 a skos:ConceptScheme ;
+  mu:uuid "b18acf1a-2a37-4b42-a549-b158d0065092" ;
+  skos:prefLabel "Gebruikersrollen"@nl .
+
+<http://themis.vlaanderen.be/id/bestuursfunctie/9d5ebfb9-3829-4b1f-a2a8-15033f7e2097> a org:Role ;
+    mu:uuid "9d5ebfb9-3829-4b1f-a2a8-15033f7e2097" ;
+    skos:inScheme thConceptScheme:b18acf1a-2a37-4b42-a549-b158d0065092 ;
+    skos:prefLabel "Vlaams Parlement" ;
+    skos:notation "Kaleidos_Parlement" ;
+    skos:definition "Secretaris" .
+
+<http://themis.vlaanderen.be/id/bestuursfunctie/cfa6ed74-bb6f-4d4c-b905-9a205be135d7> a org:Role ;
+    mu:uuid "cfa6ed74-bb6f-4d4c-b905-9a205be135d7" ;
+    skos:inScheme thConceptScheme:b18acf1a-2a37-4b42-a549-b158d0065092 ;
+    skos:prefLabel "Vlaams Parlement" ;
+    skos:notation "Kaleidos_Parlement" ;
+    skos:definition "Waarnemend Secretaris" .
```

Comment:
met deze 2 erbij hebben we nu 3 `org:Role` (1 van vorige migratie) met dezelfde `prefLabel` en `notation`.
Voor kabinet hebben we duidelijk 2 verschillende rollen gemaakt beheerder en medewerker.
Willen we dat hier ook niet?
Ik heb nog niet gekeken naar de opvolgende PR's, niet zeker hoe dat verder gebruikt wordt.